templates/lynn.html

```diff
@@ -12,21 +12,28 @@
 			<div id="logo">
 				<h1>Chat</h1>
```

Comment:
Also, I think we could write something nicer than `Chat` here . Think it over on how we could personalise this app rather than just making it generic. It has such a beautiful name too. ( This is a suggestion and can be omitted. Totally your call)